docs/sources/installation/configure-docker.md

```diff
@@ -53,6 +53,17 @@ GF_PATHS_LOGS         | /var/log/grafana
 GF_PATHS_PLUGINS      | /var/lib/grafana/plugins
 GF_PATHS_PROVISIONING | /etc/grafana/provisioning
 
+## Logging
+
+Logs in the docker container go to standard out by default, as is common in the docker world. Change this by setting a different [log mode]({{< relref "../administration/configuration.md#mode" >}}).
```

Comment:
```suggestion
Logs in theDocker container go to standard out by default, as is common in the Docker world. Change this by setting a different [log mode]({{< relref "../administration/configuration.md#mode" >}}).
```

Reply:
What is "standard out by default"? Is this a developer term I am just unfamiliar with?

Reply:
Standard output? Does that mean console? If so, we should clarify.

Reply:
`Standard out` is a technical the term for the output stream that goes to the parent process (allthough, the correct term is probably standard output). If you start grafana from the terminal the terminal will be the parent process and standard out will go to the terminal/console. But, if you run grafana via docker, docker will be the parent process and standard out will go to docker. Docker saves this information and you can access it through `docker logs`.

I guess we should change it to console to stay consistent with the config. Standard output is more correct but someone that understands what standard output means should be able to infer what we mean by console too.

Reply:
Agreed. Let's call it "console" so we keep our terminology consistent.

Reply:
@oddlittlebird You should change "theDocker" into "the Docker".